src/Installer/redist-installer/targets/Crossgen.targets

```diff
@@ -135,13 +136,6 @@
       <RemainingFolders Include="$(SdkOutputDirectory.TrimEnd('\').TrimEnd('/'))" />
     </ItemGroup>
 
-    <!-- Crossgen does not support generating symbols on Mac  -->
```

Comment:
This is an old condition which used to hold for old crossgen, but not since crossgen2. The old crossgen/ngen code was removed from coreclr.

Reply:
Yes, this can be a separate PR.

Reply:
https://github.com/dotnet/sdk/pull/43840